cpp/arcticdb/storage/azure/azure_storage.cpp

```diff
@@ -42,7 +42,8 @@ AzureStorage::AzureStorage(const LibraryPath &library_path, OpenMode mode, const
 
 Azure::Storage::Blobs::BlobClientOptions AzureStorage::get_client_options(const Config &conf) {
```

Comment:
You have to do it like [this](https://github.com/man-group/ArcticDB/pull/776/files#diff-b6fda20fb3213402c5c12984968f4f35baeaf64014bb25c402330bb4dbc9261aL43).

What makes the difference is not to make the assignment on line 48, not setting the CA path.

Reply:
Resolved